packages/opentelemetry-exporter-zipkin/package.json

```diff
@@ -4,7 +4,10 @@
   "description": "OpenTelemetry Zipkin Exporter allows the user to send collected traces to Zipkin.",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
-  "repository": "open-telemetry/opentelemetry-js",
+  "repository": "open-telemetry/opentelemetry-js",  "browser": {
```

Comment:
browser on its own line please.

Why is this a different platform configuration style than our other packages?

Reply:
Agreed for the new line.

I don't understand the issue with the "different style than the other packages", could you elaborate please ? I took [@opentelemetry/exporter-collector](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-exporter-collector/package.json) as an exemple.